v23.1/architecture/transaction-layer.md

```diff
@@ -117,6 +117,19 @@ Note that closed timestamps are valid even if the leaseholder changes, since the
 
 Closed timestamps provide the guarantees that are used to provide support for low-latency historical (stale) reads, also known as [Follower Reads](../follower-reads.html). Follower reads can be particularly useful in [multi-region deployments](../multiregion-overview.html).
 
+For long-running transactions, and generally any transaction, it can be inevitable that their timestamp is pushed, such as when they encounter a key written at a higher timestamp. When this kind of [contention](../performance-best-practices-overview.html#understanding-and-avoiding-transaction-contention) happens between transactions, the closed timestamp **may** provide some mitigation. Increasing the closed timestamp interval may reduce the likelihood of long-running transactions having their [timestamps pushed](#timestamp-cache), and therefore, reduces the likelihood of the transaction retrying. However, note that increasing the closed timestamp interval should be **carefully adjusted**, because it can have an impact on the following:
+
+- [Follower reads](../follower-reads.html): Reads will be forced to read from the leaseholder, which may affect latency/throughput.
+- [Change data capture](../change-data-capture-overview.html): There could be a lag in changefeed messages.
+- [Statistics collection](../cost-based-optimizer.html#table-statistics): Collection can place more load on the leaseholder.
+
+While increasing the closed timestamp may decrease retryable errors, it may also increase lock latencies. For example, a long-running transaction (txn 1) holds [write locks](#writing) on keys at time t=1, and txn 2 is waiting to [read](#reading) those same keys at t=1. The following scenarios may occur depending on whether the closed timestamp has been increased or not:
+
+- When txn 1 has its timestamp pushed forward by the closed timestamp, moving its writes to time t=2, txn 2 may be able to proceed and read the keys at t=1. The chances of retryable errors have increased.
+- If the closed timestamp interval is raised, long-running txn 1 is making txn 2 [wait](../performance-recipes.html#waiting-transaction), because txn 2 cannot read the keys at t=1 until txn 1 is done or pushed into the future. The chances of lock contention have increased.
```

Comment:
```suggestion
- If the closed timestamp interval is increased, `txn1` may cause `txn2`  to [wait](../performance-recipes.html#waiting-transaction), because `txn 2` cannot read the keys at `t=1` until `txn 1` is complete or is pushed into the future. The likelihood of lock contention has increased.
```

Reply:
Done!